ros2doctor/ros2doctor/api/network.py

```diff
@@ -15,19 +15,31 @@
 import os
 import sys
 
-import ifcfg
+try:
+    import ifcfg
+except ImportError:
+    sys.stderr.write('WARNING: No ifcfg module found. '
```

Comment:
Recommend using the `warnings` module https://docs.python.org/3/library/warnings.html